### ISSUE TITLE
Xbox One X doesn't show all scaned ROMs; delete app doesn't delete config files

### ISSUE THREAD

**Issue by rafaelfndev** [libretro/RetroArch#10527]
## Description

TL;DR;

- Doesn't show all scanned roms on playlist
- Delete app don't remove all config files
- [Online update > Update assets] doesn't extract and update to new interface
- Same ROMs works normally with RetroArch on PC
- Same ROMs works and appears normally on XBox One if scanned with less roms inside folder (something like 50 roms)

### Expected behavior

- Show all scanned roms
- Delete app needs to remove all config files
- Online update of assets, need to extract and apply new interface
- [Reset To Defaults] the joystick need to keep working

### Steps to reproduce the bug

1. Install the app RetroArch
2. Open RetroArch and update all with `Online Update` options
3. Copy roms to internal storage, I put all on `\\192.168.0.24\DevelopmentFiles\WindowsApps\1e4cf179-f3c2-404f-b9f3-cb2070a5aad8_1.8.5.0_x64__y40c4e9z4d7dy\Emulators\ROMs`
4. Scan path `Emulators\ROMs` with RetroArch (In my case, this folder have other folders inside with roms like "Super Nintendo, Mega Drive, Nintendo, Master System, Play Station 1" etc...
Obs: I have 1000+ roms inside folder "Mega Drive" and "Super Nintendo", maybe this can be caused errors due to the amount of roms... No one are inside .zip or .7z files, all roms are extracted with original file extension.
5. After scan all, the list doesn't recognize roms, and show up something like 60+ roms for platforms with 1000+ scanned roms.
6. I thought it was bug or corrupted file on RetroArch, so I delete all and install again, but when I try to update all again, the Assets doesn't work anymore, and I can't reset to default to try again.

[Main Menu > Configuration File > Reset To Defaults] causes the joystick to stop working, is need to end RetroArch and open it again.

Obs: on "Settings > Playlist" the size to show on playlist is only for History and Favorites, doesn't have nothing about scanned playlist.

### Version/Commit
OS version
10.0.19041.1927 (rs_xbox_release_2004.200415-0000)

Devkit type
Universal Windows App Devkit

RetroArch:
Build date: Mar 21 2020
Compiler: MSVC (1924) 64-bit
RetroArch-msvc2017-UWP_x64.appxbundle (XBox One Version)

[[Some photos of version]](https://imgur.com/a/CO8Lk40)

### Environment information

- OS: XBox One X
- Installed files via Firefox 76 / Windows 10 x64 Pro.
- I tested with Chrome and new Edge, no success.

**Comment by rafaelfndev** [comment]
Where are located of the installation configuration files? Maybe I can solve it by deleting everything that was generated in the installation and trying again. When I deleting the app through the XBox manager does not remove all files...

**Comment by rafaelfndev** [comment]
Same problem on version 1.8.7 (build date 2020/05/17). When I try to update assets, the assets.zip doesn't auto extract, when I reset to default, the joystick stops works, and delete app doesn't remove config files.

**Comment by Dragontorc** [comment]
Same issue with playlists, however it seems to be related to "manual scan" only.  When I "scan directory" with known system, playlist shows scanned roms.  When "manual scan" is made it is not.  I see LPL file is created. 

Speaking of playlists.  Are those UWP playlists encrypted?  They surely look different than playlist files on other systems I've been using RetroArch on.  When opened in text editor there are no entries like rom path in the UWP playlist files,  Everything seems to be bunch of ASCII characters.  I've hoped to scan roms on my PC and then change paths and copy LPL files to the xbox.
